### PR TITLE
update certificate-signing-requests.md to reflect https://github.com/…

### PR DESCRIPTION
**Pull request by nnmin-aws**
update certificate-signing-requests.md to reflect …kubernetes/kubernetes/pull/111660 introduced in 1.27

### CODE REVIEW

**Comment by dims** [comment]
/assign @liggitt 

Jordan, @nnmin-aws found a discrepancy PTAL

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ Kubernetes provides built-in signers that each have a well-known `signerName`:
       is not distributed by any other means.
    1. Permitted subjects - organizations are exactly `["system:nodes"]`, common name starts with "`system:node:`".
    1. Permitted x509 extensions - honors key usage extensions, forbids subjectAltName extensions and drops other extensions.
-   1. Permitted key usages - exactly `["key encipherment", "digital signature", "client auth"]`.
+   1. Permitted key usages - `["key encipherment", "digital signature", "client auth"]` if SignatureAlgorithm is RSA; `["digital signature", "client auth"]` otherwise.
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:170]: either usage is permitted, it's up to the caller to request the subset if it wants to when using RSA

```suggestion
   1. Permitted key usages - `["key encipherment", "digital signature", "client auth"]` or `["digital signature", "client auth"]`
```

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 2]
```diff
@@ -180,7 +180,7 @@ Kubernetes provides built-in signers that each have a well-known `signerName`:
    1. Permitted subjects - organizations are exactly `["system:nodes"]`, common name starts with "`system:node:`".
    1. Permitted x509 extensions - honors key usage and DNSName/IPAddress subjectAltName extensions, forbids EmailAddress and
       URI subjectAltName extensions, drops other extensions. At least one DNS or IP subjectAltName must be present.
-   1. Permitted key usages - exactly `["key encipherment", "digital signature", "server auth"]`.
+   1. Permitted key usages - `["key encipherment", "digital signature", "client auth"]` if SignatureAlgorithm is RSA; `["digital signature", "client auth"]` otherwise.
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:183]: either usage is permitted, it's up to the caller to request the subset if it wants to when using RSA

```suggestion
   1. Permitted key usages - `["key encipherment", "digital signature", "client auth"]` or `["digital signature", "client auth"]`
```

Comment (nnmin-aws) [reply content/en/docs/reference/access-authn-authz/certificate-signing-requests.md]: updated to recommendation

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md (hunk 1 above)
Comment (nnmin-aws) [reply content/en/docs/reference/access-authn-authz/certificate-signing-requests.md]: updated to recommendation

**Pull request merged**